src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+@import '../styles/variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  max-width: 166px;
```

Comment:
Consider using a variable for `max-width` if this value is reused across different components to maintain consistency and ease of maintenance.